idea-plugin/src/main/java/com/palantir/gradle/jdks/GradleJdksExternalSystemTaskNotificationListener.java

```diff
@@ -34,9 +34,6 @@ public void onStart(@NotNull ExternalSystemTaskId id, String _workingDir) {
         }
     }
 
-    @Override
-    public void onStart(@NotNull ExternalSystemTaskId _id) {}
-
     @Override
     public void onEnvironmentPrepared(@NotNull ExternalSystemTaskId _id) {}
```

Comment:
Can we remove this one too as it has a default impl (from before `onStart` had a default impl)?